src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -1104,9 +1102,23 @@ public override void Process()
     }
 
     /// <summary>
-    /// Estimator for <see cref="HashingTransformer"/> which can hash either single valued columns or vector columns. For vector columns,
-    /// it hashes each slot separately. It can hash either text values or key values.
+    /// Estimator for <see cref="HashingTransformer"/>. hash either single valued columns or vector columns. For vector columns,
+    /// it hashes each slot separately.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes, if the mapping of the hashes to the values is required. |
+    /// | Input column data type | Vector or scalars of numeric, boolean, text, [DateTime](xref: System.DateTime) and [KeyDataViewType](xref:Microsoft.ML.Data.KeyDataViewType) data types.|
```

Comment:
>[DateTime](xref: System.DateTime) [](start = 80, length = 33)

you can just use automatic links: <xref: System.DateTime> to save yourself from having to write the anchor text